docs/pages/en-us/overview.md

```diff
@@ -18,20 +18,30 @@ The Audit service provides applications with a reliable, high-performance soluti
 This documented audit trail of user and system activity has many uses.  It provides traceability and individual accountability.  Audit records can also be used for analytical purposes, including security violation detection and abnormal usage patterns.  Audit trails have legal standing and can help protect the organization with a proven record of user and system activity.  Audit records can also be exported for archival and reporting purposes.
 
 ## Introduction
-The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to Elasticsearch where they are indexed according to each tenant.
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to an endpoint such as the Audit Web Service API or directly to Elasticsearch where they are indexed according to each tenant.
```

Comment:
In my opinion I think I'd drop the use of endpoint throughout. Applications send events either directly to Elasticsearch or via the Audit Web Service.